src/manual/task-execution/workers.md

```diff
@@ -18,6 +18,9 @@ Each individual worker is identified by a `<workerGroup>/<workerId>` pair,
 identified in the task run information.  This information can be useful to
 identify misbehaving workers by examining the runs of failed tasks.
 
+You can see a list of currently-running workers, arranged by worker type, at
+https://tools.taskcluster.net/provisioners.
```

Comment:
The list of workers arranged by worker-type is in `https://tools.taskcluster.net/provisioners/:provisionerId/worker-types/:workerType`